python/taichi/lang/__init__.py

```diff
@@ -101,8 +101,8 @@
 
 When this is used, Taichi automatically picks the matching CPU backend.
 """
-timeline_clear = lambda: impl.get_runtime().prog.timeline_clear()
-timeline_save = lambda fn: impl.get_runtime().prog.timeline_save(fn)
+timeline_clear = impl.get_runtime().prog.timeline_clear
```

Comment:
`impl.get_runtime().prog` is a `Nonetype` before https://github.com/taichi-dev/taichi/blob/93c91f81aadf6745f68635a41d68896c5e1efa63/python/taichi/lang/__init__.py#L546
Therefore lambda is necessary here. We should disable the check.

Reply:
Yeah... Shall I just close this pr or keep other lambdas removed ?

Reply:
Ah you can just disable pylint here by `# pylint: disable=unneccesary-lambda` and keep others removed. It would be fine as long as passing CI. Thanks!

Reply:
Thanks! I have updated it.